mongomock/aggregate.py

```diff
@@ -984,6 +984,28 @@ def _find_matches_for_depth(query):
                 found_items.add(new_match['_id'])
         return new_matches
 
+    def _recursive_get(match, nested_fields):
```

Comment:
This function doesn't use anything apart from its parameters, so please move it at the module level.